.github/workflows/tests.yml

```diff
@@ -109,7 +103,7 @@ jobs:
       if: steps.python-files.outputs.changed_files
       run: |
         mkdir .mypy_cache/
-        mypy ${{ steps.python-files.outputs.changed_files }}
+        mypy --install-types --non-interactive ${{ steps.python-files.outputs.changed_files }} --exclude=rest_api/build/ --exclude=rest_api/test/
```

Comment:
nit: you don't need to exclude anything at this point?

Reply:
True that, but might as well leave it I guess? Better safe than sorry. 🤷